packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap

```diff
@@ -362,6 +372,16 @@ exports[`Media Player: Placeholder should render a video placeholder without dur
   cursor: pointer;
 }
 
+.emotion-1:hover .css-9vh530-StyledPlayButton,
+.emotion-1:focus .css-9vh530-StyledPlayButton {
+  background-color: #B80000;
+}
```

Comment:
These are unused duplicates of the styles below

Accounts for 5/40 of the new lines

Reply:
Closing all the placeholder snapshot comments - a fix to the styles achieved snapshot parity 👌